com.github.lainsce.notejot.json

```diff
@@ -0,0 +1,101 @@
+{
+    "app-id": "com.github.lainsce.notejot",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.lainsce.notejot",
+    "rename-icon": "com.github.lainsce.notejot",
+    "rename-appdata-file": "com.github.lainsce.notejot.appdata.xml",
+    "build-options": {
+        "env": {
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+            "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+        }
+    },
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--device=all",
+        "--socket=wayland",
+        "--filesystem=home",
```

Comment:
what do you need access to all of home for?

Reply:
For cache purposes, due to it saving to `.local/<name_of_app>/cache.json`

Reply:
Right, but you don't need all of home permission to save to a single directory path, look at the dconf section further down the manifest for an example.

And even then @TingPing will be along soon to talk about keeping this stuff within the flatpak itself ;-)

Reply:
I mean, I'm still a newbie at flatpakking stuff up, so any pointers help 👍 
I'm going to look that dconf section up and see how to do it better.